core/runtime/src/test/java/io/quarkus/runtime/configuration/UUIDPropertyTest.java

```diff
@@ -17,28 +17,24 @@
 public class UUIDPropertyTest {
 
     static ClassLoader classLoader;
-    static ConfigProviderResolver cpr;
+    static ConfigProviderResolver configProviderResolver;
 
     @BeforeAll
     public static void initConfig() {
         classLoader = Thread.currentThread().getContextClassLoader();
-        cpr = ConfigProviderResolver.instance();
+        configProviderResolver = ConfigProviderResolver.instance();
```

Comment:
I think we can drop all of the Config registration and just use SmallRyeConfig locally at each test method.